### PR TITLE
gcpohv - add support for static discovery in discovery API; fixes #621

### LINKED ISSUE #621: Stacktrace with gcp-org-hierarchy-viewer (gcpohv)
Hello,

I was looking for a way to get a quick view of my folders and projects hierarchy and found this: https://github.com/GoogleCloudPlatform/professional-services/tree/main/tools/gcp-org-hierarchy-viewer

Unfortunately, this seems to be in a broken state / does not work with the instructions provided by the README.md. I suspect that since the code provided here is ~16 months old, it is no longer compatible with the current API client version.

When I try to simply run the command against my organization ID, I get the following stacktrace:

```
$ gcpohv -o <ID>
Traceback (most recent call last):
  File "/home/user/.local/bin/gcpohv", line 8, in <module>
    sys.exit(main())
  File "/home/user/.local/bin/gcpohv_cli.py", line 269, in main
    cli = Cli(args.key_file, args.organization, args.folder, args.use_id)
  File "/home/user/.local/bin/gcpohv_cli.py", line 101, in __init__
    self._crm_v1, self._crm_v2 = setup_clients(credentials_file)
  File "/home/user/.local/bin/gcpohv_cli.py", line 62, in setup_clients
    crm_v1 = discovery.build('cloudresourcemanager', 'v1', credentials=creds)
  File "/home/user/.local/lib/python3.8/site-packages/googleapiclient/_helpers.py", line 134, in positional_wrapper
    return wrapped(*args, **kwargs)
  File "/home/user/.local/lib/python3.8/site-packages/googleapiclient/discovery.py", line 273, in build
    content = _retrieve_discovery_doc(
  File "/home/user/.local/lib/python3.8/site-packages/googleapiclient/discovery.py", line 387, in _retrieve_discovery_doc
    raise UnknownApiNameOrVersion("name: %s  version: %s" % (serviceName, version))
googleapiclient.errors.UnknownApiNameOrVersion: name: cloudresourcemanager  version: v1
```

I don't have the bandwidth to do any further investigation into, or to fix, this issue at the moment. However, my suspicion -- without digging deeper -- is that the `cloudresourcemanager v1` API is deprecated, and that the related code should be removed or wrapped in `try` statements to catch the error and move on without failing. For example here: https://github.com/GoogleCloudPlatform/professional-services/blob/bfc956a2ee8e0b6a0196bf60abfd631198a104d0/tools/gcp-org-hierarchy-viewer/gcpohv_cli.py#L62

System information:
```
$ uname -a
Linux 5.8.0-44-generic #50~20.04.1-Ubuntu SMP Wed Feb 10 21:07:30 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

$ cat /etc/issue.net
Linux Mint 20.1 Ulyssa

$ python --version
Python 3.8.5

$ pip list | grep google
google-api-core          1.26.0
google-api-python-client 2.0.0
google-auth              1.27.0
google-auth-httplib2     0.0.4
googleapis-common-protos 1.53.0
```

### PR DESCRIPTION
**Pull request by angstwad**
Adds support for static discovery when using the discovery-based API in `gcpohv` to fix #621, and upgrades the Python client to latest available w/ the requisite `static_discovery` kwarg.

### CODE REVIEW

File: tools/gcp-org-hierarchy-viewer/gcpohv_cli.py [hunk 1]
```diff
@@ -59,8 +59,14 @@ def setup_clients(credentials_file):
           Manager.
     """
     creds = setup_credentials(credentials_file)
-    crm_v1 = discovery.build('cloudresourcemanager', 'v1', credentials=creds)
-    crm_v2 = discovery.build('cloudresourcemanager', 'v2', credentials=creds)
+    crm_v1 = discovery.build('cloudresourcemanager',
+                             'v1',
+                             credentials=creds,
+                             static_discovery=True)
```
Comment (tims) [inline tools/gcp-org-hierarchy-viewer/gcpohv_cli.py:65]: Fyi, I'm not sure this does anything.

According to:
https://github.com/googleapis/google-api-python-client/blob/master/googleapiclient/discovery.py
`static_discovery` will default to `True` when `discoveryServiceUrl` is also not provided

So it was already being set to `True`.

Comment (angstwad) [reply tools/gcp-org-hierarchy-viewer/gcpohv_cli.py:65]: Yeah, upping the version is the real change.  This is just being explicit.

**Pull request merged**